Car_Code/cheme_car_competition_codebase/cheme_car_competition_codebase.ino

```diff
@@ -269,7 +264,8 @@ void loop() // Loop (main loop)
   // zAngle = mpu.getAngleZ(); // Get z-axis angle from MPU
 
   // Update kalman filters
-  kalman_filter(x_temp, p_temp, q_temp, r_temp, temperatureC, true);
+  //kalman_filter(x_temp, p_temp, q_temp, r_temp, temperatureC, true);
+  x_temp = kalman_filter_temp(temperatureC);
```

Comment:
should this be x_k_temp? x_temp doesn't seem to be referenced/created at all in your modified code apart from here. You should also initialize x_k_temp to be initTemp in setup since we'll be using x_k_temp in the conductivity sensor for temperature correction and we can't start readings off at zero deg C.